### ISSUE TITLE
ws does not work in the browser. Browser clients must use the native WebSocket object (7.0.13)

### ISSUE THREAD

**Issue by PaulMoore-PaymentsFoundry** [sclausen/ngx-mqtt#162]
- [x] **I have searched for similar issues in this repository, but couldn't find one.**
- [x] **I have read the README and have a basic understanding how angular works.**
## I'm submitting a...

- [ ] Regression (a behavior that used to work and stopped working in a new release)
- [x] Bug report  
- [ ] Feature request

## Current behavior
Throws a "ws does not work in the browser. Browser clients must use the native WebSocket object".  Does not happen in 7.0.12

## Expected behavior
Connects to MQTT using websockets.

## Environment

<pre><code>
ngx-mqtt version: 7.0.13

- Broker VerneMQ

Others:

</code></pre>

**Comment by YoDaMa** [comment]
release 4.2.1 (#1149) fixes this.

**Comment by ccravens** [comment]
I can confirm that manually updating to 4.2.1 in package-lock.json and re-running npm install solves this issue for me

**Comment by nosovk** [comment]
we use exact version in nxg-mqtt
https://github.com/sclausen/ngx-mqtt/blob/master/projects/ngx-mqtt/package.json#L12
as you see we do not use ^
Just don't add mqtt as dependancy in yours angular project. Then peed dependency will take correct version.

**Comment by markdegrootnl** [comment]
It seems the version on npm has a dependency on mqtt exact version 4.2.0.
I've downloaded the package using `npm pack ngx-mqtt` which gives me `ngx-mqtt-7.0.13.tgz`.

After extracting, these are the dependencies in package.json:
```json
"dependencies": {
  "mqtt": "4.2.0",
  "tslib": "^1.10.0"
},
```

**Comment by sclausen** [comment]
I just published [v7.0.14](https://www.npmjs.com/package/ngx-mqtt/v/7.0.14) with `mqtt@4.2.1`.

**Comment by markdegrootnl** [comment]
Great @sclausen, thanks for the quick response!
I've just tested installed the new version. The mqtt dependency is correct and the error in the browser is gone.
